app/models/plaid_investment_sync.rb

```diff
@@ -82,12 +82,15 @@ def get_security(plaid_security_id, securities)
       end
 
       return [ nil, nil ] if plaid_security.nil? || plaid_security.ticker_symbol.blank?
+      return [ nil, nil ] if plaid_security.ticker_symbol == "CUR:USD" # internally, we do not consider cash a security and track it separately
```

Comment:
Okay so this seems to be the problematic line with the tests.  In the case of a Cash security, we need to return `[ nil, plaid_cash_security ]` rather than `[nil, nil]`. 

So to get tests passing:

1. Revert this early return
2. In `securities.yml` (fixture), change both of the `exchange_mic` keys to `exchange_operating_mic`